modules/tinymce/src/core/test/ts/browser/EditorRemoveTest.ts

```diff
@@ -29,6 +34,51 @@ UnitTest.asynctest('browser.tinymce.core.EditorRemoveTest', (success, failure) =
       cRemoveEditor
     ])),
 
+    Logger.t('remove editor after stylesheets load', Chain.asStep({}, [
+      Chain.op(() => {
```

Comment:
Worth noting this won't be needed once we switch to Bedrock 10, as it catches these now and fails the test.

Reply:
Good to know. Still have the problem of making sure the test runs long enough for all the async code to resolve though.